### PR TITLE
Update documentation on access to SFTP share from a Windows client

### LINKED ISSUE #252: Remove reference to seemingly deprecated Swish tool
In our section on how to access SFTP shares from a Windows client, we link to the Swish tool among other options:
http://rockstor.com/docs/sftp/sftp.html#sftp-access-from-windows

Unfortunately, their latest release dates from June 2013 and the Github page hasn't seen much commits for a few years. It is, for instance, unsure whether it is compatible with Windows 10. As we have already quite a few other options listed, it might be preferable to simply remove our links to Swish (especially considering this is our first option listed).

On a side note, it could be interesting to add FileZilla as another option, as it is a very popular tool (like WinSCP) and is GPLv2.

@phillxnet, any thought on either of these points?

### PR DESCRIPTION
**Pull request by FroggyFlox**
Fixes #252 
@phillxnet, ready for review.

### This pull request's proposal
This pull request (PR) includes two main changes:
1. Remove mention of the deprecated tool Swish.
2. Add step-by-step description of how to access an SFTP share using the FileZilla client.

### Sphinx build
HTML files were successfully built using Sphinx v3.4.1.

<details><summary>Full Sphinx build output</summary>

```
# make html
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v3.4.1
making output directory... done
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 63 source files that are out of date
updating environment: [new config] 63 added, 0 changed, 0 removed
reading sources... [100%] windows-shadow-copy                                                                                                                                                                                                  
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] windows-shadow-copy                                                                                                                                                                                                   
generating indices... genindex done
writing additional pages... search done
copying images... [100%] change-password.gif                                                                                                                                                                                                   
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded.

The HTML pages are in _build/html.

Build finished. The HTML pages are in _build/html.
```

</details>

### Checklist
- [x] With the proposed changes no Sphinx errors or warnings are generated.
- [x] I have added my name to the AUTHORS file, if required (descending alphabetical order).

### CODE REVIEW

File: sftp/sftp.rst
Comment (phillxnet) [inline sftp/sftp.rst:227]:
```suggestion
WinSCP recommendation is for Windows users to choose the explorer interface.If
```
@FroggyFlox Sorry but I've got another little suggestion/correction here.
We are recommending that they do this, rather than that they have already done it :).

File: sftp/sftp.rst
Comment (phillxnet) [inline sftp/sftp.rst:275]:
```suggestion
below), and fill in the required information:
```
Arguable (or 'fill out' of course) so see how you feel on this one.

File: sftp/sftp.rst
Comment (phillxnet) [inline sftp/sftp.rst:281]:
```suggestion
The settings should be filled in as follows:
```
Again, as per last suggestion, this could also be 'filled out'.

**Review by phillxnet — CHANGES_REQUESTED** [review]
@FroggyFlox Thanks for getting this done. It's a major improvement and I really like the new FileZilla section. Just some minor suggestions and this one is also ready to merge and publish.

**Comment by FroggyFlox** [comment]
Thanks again for catching all these, @phillxnet !

**Comment by phillxnet** [comment]
@FroggyFlox Cheers for the rapid maintenance. As I see you have now reviewed all pending reviews in this repo I'll hold off publishing until they are merged.
Cheers. Merging as previously discussed.

**Comment by phillxnet** [comment]
@FroggyFlox Cheers. Now published and much appreciated.

**Pull request merged**
